drag_and_drop_v2/public/js/translations/eo/text.js

```diff
@@ -16,181 +16,6 @@
 
   django.catalog = django.catalog || {};
   
-  var newcatalog = {
```

Comment:
@eLRuLL  This shoudn't be removed. The `eo` language code here is used as a ["dummy" translation](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/271/files#diff-d0a28b90104be3005d4da328f9991ce959800fcedb329cc503f89057e181d15cR88), for testing.

`eo` is what [edx uses](https://github.com/edx/edx-platform/blob/b4614d75f0eb56861388ca31ae8cfa840a0c9374/conf/locale/config.yaml#L95-L98) too, but it's unfortunate, since `eo` is `Esperanto`.

You could change the `drag_and_drop_v2/translations/config.yaml` file to use the word "dummy" like we do in LX.

But either way, you need to set up a virtualenv and run these commands to reinstate the dummy translation files.

```
pip install -r requirements.txt
make build_dummy_translations
```